app/routers/case_information.py

```diff
@@ -18,11 +23,11 @@
 
 
 @router.get("/{case_id}", tags=["cases"])
-async def read_case(case_id: UUID, session: Session = Depends(get_session)) -> Case:
+async def read_case(case_id: UUID, session: Session = Depends(get_session)):
     case: Case | None = session.get(Case, case_id)
     if not case:
         raise HTTPException(status_code=404, detail="Case not found")
-    return case
+    return CaseResponse.from_orm(case)
```

Comment:
What's the difference between returning `CaseResponse.from_orm(case)` and `case`

Reply:
From ORM reads the attributes of a class as defined by another class. It acts recursively https://docs.pydantic.dev/1.10/usage/models/#orm-mode-aka-arbitrary-class-instances

Reply:
If we just go for case it only returns the created at and id